### PR TITLE
[Purify] remove upsells and branding from pre-fork

### PR DESCRIPTION
**Pull request by kavilla**
### Description
Left-over upsells and branding that were missed in previous
updates since they are not easy to catch.

Signed-off-by: Kawika Avilla <kavilla414@gmail.com>
 
### Issues Resolved
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/359
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/388
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/384
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/data/public/ui/query_string_input/language_switcher.tsx [hunk 1]
```diff
@@ -104,8 +104,8 @@ export function QueryLanguageSwitcher(props: Props) {
             <FormattedMessage
               id="data.query.queryBar.syntaxOptionsDescription"
               defaultMessage="The {docsLink} (DQL) offers a simplified query
-              syntax and support for scripted fields. DQL also provides autocomplete if you have
-              a Basic license or above. If you turn off DQL, OpenSearch Dashboards uses Lucene."
+              syntax and support for scripted fields. If you turn off DQL,
```
Comment (tmarkley) [inline src/plugins/data/public/ui/query_string_input/language_switcher.tsx:107]: Was autocomplete removed?

File: src/plugins/maps_legacy/public/map/map_messages.js [hunk 2]
```diff
@@ -110,39 +110,17 @@ export const createZoomWarningMsg = (function () {
             <FormattedMessage
               id="maps_legacy.opensearchDashboardsMap.zoomWarning"
               defaultMessage="You've reached the maximum number of zoom
-              levels. To zoom all the way in, upgrade to the
-              {defaultDistribution} of OpenSearch and OpenSearch Dashboards. You'll get
-              access to additional zoom levels for free through the {ems}.
-              Or, you can configure your own map server. Please go to
-              { wms } or { configSettings} for more information."
-              // TODO: [RENAMEME] Need valid URLs
+              levels. Please go to { wms } for more information."
```
Comment (tmarkley) [inline src/plugins/maps_legacy/public/map/map_messages.js:113]: Do we need to leave the instructions about configuring your own map server? Is it still possible to access additional zoom levels through other means?

File: src/plugins/tile_map/public/tile_map_type.js [hunk 3]
```diff
@@ -38,15 +38,13 @@ import { createTileMapVisualization } from './tile_map_visualization';
...
 import { truncatedColorSchemas } from '../../charts/public';
-import { getDeprecationMessage } from './get_deprecation_message';
 
 export function createTileMapTypeDefinition(dependencies) {
   const CoordinateMapsVisualization = createTileMapVisualization(dependencies);
   const { uiSettings, getServiceSettings } = dependencies;
 
   return {
     name: 'tile_map',
-    getInfoMessage: getDeprecationMessage,
```
Comment (tmarkley) [inline src/plugins/tile_map/public/tile_map_type.js:49]: Is this prop referenced elsewhere?

File: src/plugins/region_map/public/region_map_type.js [hunk 4]
```diff
@@ -37,15 +37,13 @@ import { RegionMapOptions } from './components/region_map_options';
...
 import { ORIGIN } from '../../maps_legacy/public';
-import { getDeprecationMessage } from './get_deprecation_message';
 
 export function createRegionMapTypeDefinition(dependencies) {
   const { uiSettings, regionmapsConfig, getServiceSettings } = dependencies;
   const visualization = createRegionMapVisualization(dependencies);
 
   return {
     name: 'region_map',
-    getInfoMessage: getDeprecationMessage,
```
Comment (tmarkley) [inline src/plugins/region_map/public/region_map_type.js:48]: Is this prop referenced elsewhere?

File: src/plugins/tile_map/public/tile_map_type.js (hunk 3 above)
Comment (kavilla) [reply src/plugins/tile_map/public/tile_map_type.js:49]: This is optional on the type visType. So removing it here satisfies the problem we are seeing.

File: src/plugins/maps_legacy/public/map/map_messages.js (hunk 2 above)
Comment (kavilla) [reply src/plugins/maps_legacy/public/map/map_messages.js:113]: This is a good point then can use their own.

File: src/plugins/data/public/ui/query_string_input/language_switcher.tsx (hunk 1 above)
Comment (tmarkley) [reply src/plugins/data/public/ui/query_string_input/language_switcher.tsx:107]: I was asking specifically about the autocomplete feature. Does DQL not provide autocomplete after the fork?

Comment (kavilla) [reply src/plugins/data/public/ui/query_string_input/language_switcher.tsx:107]: Yeah it's part of the x-pack stuff, seems like it's been a feature request to implement a substitute for a while https://github.com/opendistro-for-elasticsearch/sample-code/issues/131

Comment (tmarkley) [reply src/plugins/data/public/ui/query_string_input/language_switcher.tsx:107]: Okay, good to know.

**Review by tmarkley — APPROVED** [review]

**Review by ananzh — APPROVED** [review]
LGTM

**Pull request merged**